Library/Formula/bitrise.rb

```diff
@@ -0,0 +1,26 @@
+require "language/go"
+
+class Bitrise < Formula
+  desc "Command-line automation tool"
+  homepage "https://github.com/bitrise-io/bitrise"
+  url "https://github.com/bitrise-io/bitrise/archive/0.9.7.tar.gz"
+  sha256 "cecd0cd112941c6b6f9d6cd334ea6a31aef1123af89e17c3950fbc8e8dbe5de2"
+
+  depends_on "go" => :build
+
+  def install
+    mkdir_p buildpath/"src/github.com/bitrise-io"
+    ln_s buildpath, buildpath/"src/github.com/bitrise-io/bitrise"
+
+    ENV["GOPATH"] = "#{buildpath}/Godeps/_workspace:#{buildpath}"
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", "_bin/bitrise"
+
+    bin.install "_bin/bitrise"
+  end
+
+  test do
+    output = shell_output(bin/"bitrise --version")
```

Comment:
We can do that, but it's a bit more complicated.

A sample `bitrise.yml` like this would be enough:

```
format_version: 0.9.8
default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib
app:
  envs: []
workflows:
  test:
    title: Workflow for master branch
    envs: []
    steps:
    - script:
        title: Test
        inputs:
        - content: |-
            #!/bin/bash
            set -e
            echo "Test - OK"
```

But to run this you first have to call a `bitrise setup`, because bitrise depends on two other tools to run these configurations (https://github.com/bitrise-io/envman/ and https://github.com/bitrise-io/stepman/).

You could use `bitrise` without `stepman` but in that case it's not possible to use a Step Library manager (`stepman`), which means that the steps used in `bitrise.yml` have to be included relative to the `bitrise.yml`, similar to the setup you can find here: https://github.com/bitrise-io/bitrise/tree/master/_examples/tutorials/steps-and-workflows - which would make the test more complicated, a bit too complicated IMO.